packages/lesswrong/components/comments/CommentsItem/CommentBody.tsx

```diff
@@ -87,23 +112,49 @@ const CommentBody = ({
     highlights = votingSystem.getCommentHighlights({comment, voteProps});
   }
 
-  const contentBody = <ContentStyles contentType={contentType} className={classes.root}>
-    <ContentItemBody
-      ref={commentBodyRef ?? undefined}
-      className={bodyClasses}
-      dangerouslySetInnerHTML={{__html: innerHtml }}
-      description={`comment ${comment._id}`}
-      nofollow={(comment.user?.karma || 0) < nofollowKarmaThreshold.get()}
-      replacedSubstrings={highlights}
-    />
-  </ContentStyles>
+  const renderContent = (content: string) => (
+    <ContentStyles contentType={contentType} className={classes.root}>
+      <ContentItemBody
+        ref={commentBodyRef ?? undefined}
+        className={bodyClasses}
+        dangerouslySetInnerHTML={{__html: content}}
+        description={`comment ${comment._id}`}
+        nofollow={(comment.user?.karma || 0) < nofollowKarmaThreshold.get()}
+        replacedSubstrings={highlights}
+      />
+    </ContentStyles>
+  );
+
+  const tabContent = ({activePotentialComment, potentialComments, setActivePotentialComment, doppelCommentVote}:
+    { activePotentialComment: number,
+      potentialComments: {content: string}[],
+      setActivePotentialComment: React.Dispatch<SetStateAction<number>>,
+      doppelCommentVote: React.ReactNode
+    }) => (
+    <>
+      <div className={classes.doppelQuizIntro}>
+        <h3 className={classes.doppelQuizHeader}>Which of these 'comments' is the real one, and which generated by LM?</h3>
+        {(comment.contents?.plaintextMainText?.length ?? 0) > 3e3 ? doppelCommentVote : null}
```

Comment:
I don't think 3000 is large enough to deserve scientific notation.  (You can do `3_000` if you want the readability benefit.)

Reply:
(Also this expression is too complicated, plz extract & factor out into at least 2 pieces)

Reply:
what's the cost?